docs/conf.py

```diff
@@ -89,9 +89,18 @@
     ],
     # Configure secondary (right) side bar
     "show_toc_level": 3,                     # Show all subsections of notebooks
+    "show_nav_level": 2,
     "secondary_sidebar_items": ["page-toc"]  # Omit 'show source' link that that shows notebook in json format
 }
 
+html_sidebars = {
+    "community/index": [
+        "sidebar-nav-bs",
+    ],
+    "help/index": [
+        "sidebar-nav-bs",
```

Comment:
```suggestion
```
delete it would remove the sidebar from help/index, which might be good for this example since it is empty

Reply:
We would have to wait for #59 to decide on this, then - right?